service/service.go

```diff
@@ -70,14 +79,17 @@ func Run(ctx context.Context, cfg *config.Config, serviceList *ExternalServiceLi
 			return nil, err
 		}
 
-		elasticHTTPClient = dphttp.NewClientWithTransport(awsSignerRT)
+		esClient, esClientErr = dpEs.NewClient(dpEsClient.Config{
+			ClientLib: dpEsClient.GoElasticV710,
+			Address:   cfg.ElasticSearchAPIURL,
+			Transport: awsSignerRT,
+		})
+		if esClientErr != nil {
+			log.Fatal(ctx, "Failed to create dp-elasticsearch client", esClientErr)
+		}
+		transformerClient = transformer.New()
```

Comment:
Does it not make sense to for transformer.New() to take in a variable to define if it is legacy transformer or 7.10 one, like queryBuilder is set to `query.NewQueryBuilder(cfg.ElasticVersion710)` on line 94